### DISCUSSION TITLE
Some parts are not visible in the rotation movement

### DISCUSSION THREAD

**Discussion by bibasaktanesi** [google/model-viewer#3798] [Q&A]
Hey hi hope you are well!

In my Modelviewer work, the parts I show in the picture become invisible while rotating. How can I fix this problem?
![gözükmeyen parçalar](https://user-images.githubusercontent.com/110672110/189669317-6b388881-91e6-40a9-8ad0-0119296cfbfb.PNG)

1 right
2 does not appear

**Comment by elalish** [comment]
This looks like a z-order problem for transparent materials. It shouldn't be a problem if you have only one transparent mesh; is it possible you have `BLEND` or `MASK` mode set on objects that should be `OPAQUE`?
